main.py

```diff
@@ -1,26 +1,113 @@
+import requests
 import functools
 import os
+import subprocess
 import random
 import sys
 import time
 
+import speech_recognition as sr
 from faker import Faker
 from selenium import webdriver
 from selenium.webdriver.support.ui import Select
+from selenium.webdriver.common.keys import Keys
 
 from constants.common import *
+from constants.fileNames import *
+from constants.classNames import *
 from constants.elementIds import *
 from constants.email import *
 from constants.location import *
 from constants.urls import *
 from constants.xPaths import *
 
+os.environ["PATH"] += ":/usr/local/bin" # Adds /usr/local/bin to my path which is where my ffmpeg is stored
+
 fake = Faker()
 chromedriver_location = CHROMEDRIVER_PATH
 # Change default in module for print to flush
 # https://stackoverflow.com/questions/230751/how-can-i-flush-the-output-of-the-print-function-unbuffer-python-output#:~:text=Changing%20the%20default%20in%20one%20module%20to%20flush%3DTrue
 print = functools.partial(print, flush=True)
 
+r = sr.Recognizer()
+
+def audioToText(mp3Path):
+    # deletes old file
+    try:
+        os.remove(CAPTCHA_WAV_FILENAME)
+    except FileNotFoundError:
+        pass
+    # convert wav to mp3                                                            
+    subprocess.run(f"ffmpeg -i {mp3Path} {CAPTCHA_WAV_FILENAME}", shell=True, timeout=5)
+
+    with sr.AudioFile(CAPTCHA_WAV_FILENAME) as source:
+        audio_text = r.listen(source)
+        try:
+            text = r.recognize_google(audio_text)
+            print('Converting audio transcripts into text ...')
+            return(text)     
+        except Exception as e:
+            print(e)
+            print('Sorry.. run again...')
+
+def saveFile(content,filename):
+    with open(filename, "wb") as handle:
+        for data in content.iter_content():
+            handle.write(data)
+# END TEST
+
+def solveCaptcha(driver):
+    # Logic to click through the reCaptcha to the Audio Challenge, download the challenge mp3 file, run it through the audioToText function, and send answer
+    googleClass = driver.find_elements_by_class_name(CAPTCHA_BOX)[0]
+    time.sleep(2)
+    outeriframe = googleClass.find_element_by_tag_name('iframe')
+    time.sleep(1)
+    outeriframe.click()
+    time.sleep(2)
+    allIframesLen = driver.find_elements_by_tag_name('iframe')
```

Comment:
"allIFrames" seems like a more accurate name for this variable, unless you want to just save the len() of this directly and skip it on line 71

Reply:
Yeah you are correct there